--- test/js/make.sh 2022-10-04 11:19:40.627801338 +0200
+++ test/js/make-new.sh 2022-10-04 11:20:34.745998393 +0200
@@ -33,7 +33,7 @@
 	filename=${file%.*}
 	echo $($RUN_JS "$filename.js" 2>&1) > "$filename.node.out"
 	echo $($RUN_MI "$filename.mc" 2>&1) > "$filename.mi.out"
-	diff_output=$(diff -y "$filename.node.out" "$filename.mi.out" 2>&1)
+	diff_output=$(diff "$filename.node.out" "$filename.mi.out" 2>&1)
   	exit_code=$?
 	clean_out
 	set -e
